doc/classes/EditorSettings.xml

```diff
@@ -1537,7 +1558,44 @@
 		</member>
 		<member name="text_editor/theme/highlighting/function_color" type="Color" setter="" getter="">
 			The script editor's function call color.
-			[b]Note:[/b] When using the GDScript syntax highlighter, this is replaced by the function definition color configured in the syntax theme for function definitions (e.g. [code]func _ready():[/code]).
+			[b]Note:[/b] When using the GDScript syntax highlighter, this is only used when calling some functions since function definitions and global functions have their own colors [member text_editor/theme/highlighting/gdscript/function_definition_color] and [member text_editor/theme/highlighting/gdscript/global_function_color].
+		</member>
+		<member name="text_editor/theme/highlighting/gdscript/annotation_color" type="Color" setter="" getter="">
+			The GDScript syntax highlighter text color for annotations.
+			[codeblock]
+			@export
+			[/codeblock]
```

Comment:
I don't think this kind of code block is particularly necessary for a bunch of reasons:
- In the editor, it's not hard to confront between an open tab and the editor settings. Looking at the tooltip every time is comparatively cumbersome.
- It takes a lot of space for an example that can be written in the previous line.
- The need to write this speaks for a lack of a better way to visualize it with, say, an editor section specialized in setting up a custom theme.